packages-backend/loggers/src/formatters/rewrite-fields.ts

```diff
@@ -5,35 +5,35 @@ import getIn from 'lodash/get';
 import setIn from 'lodash/set';
 import unsetIn from 'lodash/unset';
 
-type TRewriteField = {
+type TRewriteField<FieldValue> = {
   from: string;
   to: string;
-  replaceValue?: (value: string) => unknown;
+  replaceValue?: (value: FieldValue) => unknown;
```

Comment:
The field can now be typed using a generic.

Reply:
nice!